Tema6/src/main/java/SetString.java

```diff
@@ -0,0 +1,54 @@
+import java.util.Comparator;
+import java.util.HashSet;
+import java.util.Random;
+import java.util.Set;
+
+public class SetString {
```

Comment:
StringOperations is a better name for this (just because it contains a Set it doesn't mean it can't contain something else)